### ISSUE TITLE
Cannot find module 'node:path' with new version

### ISSUE THREAD

**Issue by DarthVan** [syJSdev/rollup-plugin-copy-merge#8]
Hi!
Before new version of the plugin it workd fine, but now

```
var path = require('node:path');
var util = require('node:util');
var os = require('node:os');
```

calls a bug "Error: Cannot find module 'node:path'" in my VSCode
Changing `var path = require('node:path'); → var path = require('path');` and other 2 rows fixes my issue.

Node version: 14.15.5
npm: 6.14.17
VSCode: 1.68.0

Thanks!

**Comment by syJSdev** [comment]
fixed by #9

**Comment by DarthVan** [comment]
Thanks! ☺
